README.md

```diff
@@ -107,6 +107,8 @@ proxy requests. The following options are supported:
   Example: `["cookie"]`
 * dictionary of lowercase strings `setHeaders` - Set headers for the request (overwrites existing ones).  
   Example: `{"x-powered-by": "CORS Anywhere"}`
+* number `maxAge` - If set, an Access-Control-Max-Age header with this value (in seconds) will be added.
```

Comment:
Add two spaces at the end, then markdown will show a forced line break.

And use "response header" instead of just "header".